### DISCUSSION TITLE
[Jetstream] Tag updated

### DISCUSSION THREAD

**Discussion by caouecs** [Laravel-Lang/common#42] [Q&A]
* [x] ar
 * [x]  de
 * [ ]  it
 * [x]  lt
 * [ ]  pl
 * [ ]  pt
 * [ ]  pt_BR
 * [x] ru
 * [ ]  sr_Cyrillic
 * [ ]  sr_Latin

**Comment by andrey-helldar** [comment]
ru: https://github.com/Laravel-Lang/lang/pull/1427

**Comment by andrey-helldar** [comment]
@Laravel-Lang/laravel-lang, guys...

**Comment by luisprmat** [comment]
Jetstream 2.0 will be released very soon with new translations for the team invitations module.
I guess we'll wait for it to be officially released to make our modifications

**Reply by andrey-helldar** [reply]
I think yes, you have to wait.

**Comment by mohamedsabil83** [comment]
ar: #1431

**Comment by sotten** [comment]
de: #1710
